packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -338,6 +340,29 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
     return transformedComponents;
   }
 
+  hasDynamicComponents(placeholderData: (ComponentRendering | HtmlElementRendering)[]): boolean {
+    return placeholderData.some((rendering) => {
+      const componentRendering = rendering as ComponentRendering;
+
+      if (!componentRendering?.componentName) return false;
+
+      const component = this.getComponentForRendering(componentRendering);
+
+      const isDynamicComponent = !!(component as JssComponentType)?.render?.preload;
+
+      const isByocWrapper = componentRendering.componentName === BYOC_WRAPPER_RENDERING_NAME;
+
+      const nestedPlaceholders = componentRendering?.placeholders;
+      if (nestedPlaceholders) {
```

Comment:
this will disable error boundary for **all** the nested components that eventually contain byoc, both dynamic and not? It would be a regression then.
Does the issue occur still without this part?
As an alternative, could you we wrap <FEAAS.ExternalComponent> in <Suspense> in `BYOCComponent.tsx`?

Reply:
without this part it will definitely occur since it will then wrap all components with suspense.
as discussed the latter suggested approach will bring us back to the same problem.